project/npda/templates/npda/npdauser_form.html

```diff
@@ -20,6 +20,12 @@
                 </select>
               {% elif field.field.widget|is_textinput %}
                 <input type="text" id="{{ field.id_for_label }}" name="{{ field.html_name }}" {% if field.value %} value="{{ field.value }}" {% endif %} class="input rcpch-input-text">
+              {% elif field.field.widget|is_checkbox and request.user.is_superuser and field.id_for_label == 'id_is_superuser' or field.id_for_label == 'id_is_staff' %}
+                {{ field }}
+              {% elif field.field.widget|is_checkbox and request.user.is_rcpch_staff and field.id_for_label == 'id_is_rcpch_staff' %}
+                {{ field }}
+              {% elif field.field.widget|is_checkbox and request.user.is_rcpch_audit_team_member and field.id_for_label == 'id_is_rcpch_audit_team_member' %}
+                {{ field }}
               {% else %}
                 {{ field }}
```

Comment:
We need to remove this else statement as I believe this is what is causing the checkboxes to render for those people who it shouldn't render for. An elif statement above checks if a user is a superuser AND if the field.id_for_label is 'is_superuser', and if this resolves to False, then the else statement will load in the field anyway at the end.

We will just have to explicitly load the email field (as this is the only field not of type select, textinput or checkbox and thus would only be rendered by the problematic else statement unless explicitly called)